apkid/rules/elf/packers.yara

```diff
@@ -289,3 +289,28 @@ rule upx_unknown_version_unmodified : packer
     upx_unmodified and
     not upx_compressed_apk
 }
+
+rule ollvm_obfuscator : obfuscator
+{
+  meta:
+    description = "Obfuscator-LLVM native"
+
+  strings:
+    $clang_version = "Obfuscator-LLVM clang version"
+    $based_on = "(based on Obfuscator-LLVM"
+
+  condition:
+    all of them
+}
+
+rule arxan_native : obfuscator
+{
+  meta:
+    description = "Arxan native"
```

Comment:
This should use the same name as the DEX version, which should just be the name of the product and _possibly_ a version range, if possible, in general. It's extra information that should be implied by the file type.